src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -41,13 +41,13 @@
 namespace Microsoft.IdentityModel.JsonWebTokens
 {
     /// <summary>
-    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens. 
+    /// A <see cref="SecurityTokenHandler"/> designed for creating and validating Json Web Tokens.
     /// See: http://tools.ietf.org/html/rfc7519 and http://www.rfc-editor.org/info/rfc7515.
     /// </summary>
     public class JsonWebTokenHandler : TokenHandler
     {
         /// <summary>
-        /// Gets the Base64Url encoded string representation of the following JWT header: 
+        /// Gets the Base64Url encoded string representation of the following JWT header:
```

Comment:
Should be a period and the end instead of at ':'.

Reply:
I think this is not a typo but just multi-line comment wrapped after ':'. See how Visual Studio renders it: 
![image](https://user-images.githubusercontent.com/9095515/141183523-aad4a2db-766a-4487-98e4-5e8d3090b131.png)
Should I perform the change that you suggested anyway?

Reply:
i would leave it as you have it @lechu445  w/the `:`